src/lib/isDivisibleBy.js

```diff
@@ -1,7 +1,9 @@
 import assertString from './util/assertString';
+import assertNumber from './util/assertNumber';
 import toFloat from './toFloat';
 
 export default function isDivisibleBy(str, num) {
   assertString(str);
+  assertNumber(num);
```

Comment:
`parseInt` returns `NaN` if the value cannot be parsed (e.g. because it is a string) and does not throw. IMHO that is the correct behaviour the function should have since `10 % NaN === 0 => false`.

Reply:
thank you